ignite/services/plugin/plugin_test.go

```diff
@@ -363,7 +363,7 @@ func TestPluginLoad(t *testing.T) {
 			manifest, err := p.Interface.Manifest(ctx)
 			require.NoError(err)
 			assert.Equal(p.name, manifest.Name)
-			assert.NoError(p.Interface.Execute(ctx, &ExecutedCommand{}, clientAPI))
+			assert.NoError(p.Interface.Execute(ctx, &ExecutedCommand{OsArgs: []string{"ignite", p.name}}, clientAPI))
```

Comment:
Why are the `OsArgs` required in the test? Is there an issue or failure somewhere if it's not present?

Reply:
Removing this causes a panic and that's because by default we assume that `len(OsArgs) > 1`
https://github.com/ignite/cli/blob/feat/improve-app-scaffolding/ignite/services/plugin/template/main.go.plush#L30